plugins/secretstores/gdchauth/README.md

```diff
@@ -0,0 +1,36 @@
+# Docker Secrets Secret-Store Plugin
+
+The `gdchauth` plugin allows to fetch token for a service account file and STS audience.
```

Comment:
Wrong title, and maybe also provide a link to additional information about where to get that file or audience info.